src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ContextActionCell.cs

```diff
@@ -33,19 +33,19 @@ static ContextActionsCell()
 		{
 			var rect = new RectangleF(0, 0, 1, 1);
 			var size = rect.Size;
-
-			UIGraphics.BeginImageContext(size);
-			var context = UIGraphics.GetCurrentContext();
-			context.SetFillColor(Microsoft.Maui.Platform.ColorExtensions.Red.CGColor);
-			context.FillRect(rect);
-			DestructiveBackground = UIGraphics.GetImageFromCurrentImageContext();
-
-			context.SetFillColor(Microsoft.Maui.Platform.ColorExtensions.LightGray.CGColor);
-			context.FillRect(rect);
-
-			NormalBackground = UIGraphics.GetImageFromCurrentImageContext();
-
-			context.Dispose();
+			using var renderer = new UIGraphicsImageRenderer(size);
+			DestructiveBackground = renderer.CreateImage((UIGraphicsImageRendererContext ctx) =>
+			{
+				var context = ctx.CGContext;
```

Comment:
This is almost duplicated two times, can create a method and reuse it. Should be something like:

```
private void FillRect(UIGraphicsImageRendererContext ctx, CGRect rect, CGColor color)
{
    var context = ctx.CGContext;
    context.SetFillColor(color);
    context.FillRect(rect);
}
```

```
using var renderer = new UIGraphicsImageRenderer(size);
DestructiveBackground = renderer.CreateImage(ctx =>
    FillRect(ctx, rect, Microsoft.Maui.Platform.ColorExtensions.Red.CGColor));
NormalBackground = renderer.CreateImage(ctx =>
    FillRect(ctx, rect, Microsoft.Maui.Platform.ColorExtensions.LightGray.CGColor));
```

Reply:
@jsuarezruiz, I have modified it.